theme/src/resources/templates/article_page.hbs

```diff
@@ -129,4 +131,38 @@
 		},
 		document.getElementById('sidenavTree')
 	);
+
+	window.onload = function() {
+		const articleContent = document.querySelector('.article-content');
+
+		const headingsNodeList = document.querySelectorAll('.article-body h2');
+
+		if (articleContent && headingsNodeList.length > 0) {
```

Comment:
I don't think we need to check for `> 0` since if there is a value that would equate to truthy.

Reply:
Oops, you're right. Let me make the change.